components/script/dom/webglrenderingcontext.rs

```diff
@@ -4227,7 +4228,7 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
             handle_potential_webgl_error!(self, self.validate_ownership(texture), return);
         }
 
-        if target != constants::FRAMEBUFFER {
+        if target != gl::UNSIGNED_INT_24_8 {
```

Comment:
This also doesn't make sense. The spec says ` The framebufferTexture2D entry point is extended to accept the target parameter DEPTH_ATTACHMENT and DEPTH_STENCIL_ATTACHMENT`, which is not what's being checked.